index.bs

```diff
@@ -1283,22 +1283,22 @@ defined in [[!WebCryptoAPI]].
 The API defined in this specification implies a specific abstract functional model for an [=authenticator=]. This section
 describes the authenticator model.
 
-Client platforms may implement and expose this abstract model in any way desired. However, the behavior of the client's Web
+Client platforms MAY implement and expose this abstract model in any way desired. However, the behavior of the client's Web
 Authentication API implementation, when operating on the authenticators supported by that platform, MUST be indistinguishable
 from the behavior specified in [[#api]].
 
-For authenticators, this model defines the logical operations that they must support, and the data formats that they expose to
+For authenticators, this model defines the logical operations that they MUST support, and the data formats that they expose to
 the client and the [=[RP]=]. However, it does not define the details of how authenticators communicate with the client platform,
-unless they are required for interoperability with [RPS]. For instance, this abstract model does not define protocols for
+unless they are necessary for interoperability with [RPS]. For instance, this abstract model does not define protocols for
 connecting authenticators to clients over transports such as USB or NFC. Similarly, this abstract model does not define specific
 error codes or methods of returning them; however, it does define error behavior in terms of the needs of the client. Therefore,
-specific error codes are mentioned as a means of showing which error conditions must be distinguishable (or not) from each other
+specific error codes are mentioned as a means of showing which error conditions MUST be distinguishable (or not) from each other
```

Comment:
This is descriptive - not normative language. Please back this individual change out.

Reply:
done.